docs/src/parameter_distributions.md

```diff
@@ -361,28 +357,108 @@ end
 gif(anim_bounded, "anim_bounded.gif", fps = 10) # hide
 ```
 
-The following generates the transformed `Normal(0.5, 1)` distribution
 
+
+## Full Examples
+
+### Simple example - 1
+
+Task: We wish to create a prior for a one-dimensional parameter. Our problem dictates that this parameter is bounded between 0 and 1; domain knowledge leads us to expect it should be around 0.7. The parameter is called `point_seven`.
+
+In this section we show how to specify this with the `constrained_gaussian` constructor, while in the following section we show how to specify the prior "manually" with the general constructor.
+
+We're told that the prior mean is 0.7; we choose a prior standard deviation of 0.15 to be sufficiently wide without putting too much mass at the upper bound. The constructor is then
 ```julia
-using EnsembleKalmanProcesses.ParameterDistributions
+prior = constrained_gaussian("point_seven", 0.7, 0.15, 0.0, 1.0)
+```
+
+```@setup example_one
+# instead of importing ParameterDistributions & dependencies to call constructor,
+# which would make docs build longer and more fragile, simply hard-code Normal()
+# parameters found by constrained_gaussian constructor
+
 using Distributions
+using Plots
+Plots.default(lw=2)
 
-param_dict = Dict(
-"distribution" => Parameterized(Normal(0.5, 1)),
-"constraint" => bounded(5, 10),
-"name" => "bounded_parameter",
-)
+N = 50
+x_eval = collect(-5:6/400:5)
 
-prior = ParameterDistribution(param_dict)
+#bounded in [0.0, 1.0]
+transform_unconstrained_to_constrained(x) = 1.0 - 1.0 / (exp(x) + 1)
+dist= pdf.(Normal(0.957711, 0.78507), x_eval)
+constrained_x_eval = transform_unconstrained_to_constrained.(x_eval)
+
+p2 = plot(constrained_x_eval, dist) 
+vline!([0.7]) 
+title!("Prior pdf")
+```
+
+The pdf of the constructed prior distribution (in the physical, constrained space) looks like:
+```@example example_one
+p = plot(p2, legend=false, size = (450, 450)) #hide
+```
+
+### Simple example - 2
```

Comment:
It would be great if these "simple examples" were moved above all the constraints discussion

Reply:
Addressed in 5ef7856